test/unit/pipeline/test_train_model.py

```diff
@@ -155,6 +157,21 @@ def test_train_model_pipeline_young_model(self, serializer_mock, pipeline_mock):
         )
         self.assertFalse(pipeline_mock.called)
 
+    @patch("openstf.pipeline.train_model.validation")
+    def test_train_model_InputDataInsufficientError(self, validation_is_data_sufficient_mock):
+        validation_is_data_sufficient_mock.is_data_sufficient.return_value = False
+        with self.assertRaises(InputDataInsufficientError):
+            train_model_pipeline_core(pj=self.pj, input_data=self.train_input)
+
+    def test_train_model_InputDataWrongColumnOrderError(self):
+        input_data = self.train_input
+        input_data["load2"] = input_data["load"]
+        input_data = input_data.drop("load", axis=1)
+        input_data["load"] = input_data["load2"]
+        input_data = input_data.drop("load2", axis=1)
```

Comment:
You could help a (future) colleague by adding a small comment of what this code does (I think it moves the first column to the end?).
Also, perhaps you can make it a bit shorter:
`input_data = input_data.iloc[:,1:].merge(input_data.iloc[:,[0]], left_index=True, right_index=True)` 
(untested code snippit)